source/features/view-markdown-source.tsx

```diff
@@ -95,11 +95,7 @@ async function init(): Promise<void> {
 	delegate(document, '.rgh-md-source:not(.selected)', 'click', showSource);
 	delegate(document, '.rgh-md-rendered:not(.selected)', 'click', showRendered);
 
-	const fileButtons =
-		select('.repository-content .Box-header.flex-md-items-center .d-flex') ??
-		// Pre "Repository refresh" layout
-		select('.repository-content .Box-header .d-flex')!;
-	fileButtons.prepend(
+	select('.repository-content readme-toc .Box-header .d-flex')!.prepend(
```

Comment:
This selector has 2 elements:

<img width="532" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/115049950-76b29500-9f05-11eb-8d7b-305779ac23b2.png">


Let's use the new suggested way to find items more reliably. Maybe 
```suggestion
	select('#raw-url')!.closest('.d-flex')!.prepend(
```

Reply:
Successfully tested this change on Firefox.